UHERO/udaman
# Add selected DBEDT series to COH portal (UA-951)

A fair amount of refactoring here, around crudding and relating of data lists/measurements/series. Potential cascading refactor in `nta_upload.rb` avoided (in favor of just a comment) so as not to have to retest all of that functionality.

`Measurement#edit` screen has big change: list of series moved to the top and excess headings, etc removed, because I feel it is much more useable this way.

### Testing
(I added a category called `Testtchild` on stage with stuff in it)
- Add a new category with new data list to COH portal (or maybe just edit existing ones)
- Add a DBEDTCOH measurement to data list
- Check API output for URIs like the following (changing params as needed):
-- `v1/search/series?q=school&u=coh`
-- `/v1/search?q=school&u=coh`
-- `/v1/package/search?q=school&u=coh` (uses default geo/freq from included search summary)
-- `/v1/package/search?q=school&u=coh&geo=KAU&freq=A`
-- `/v1/category/series?id=<your cat id>`

I am getting responses from API as expected, but when I test with the data portal front end, some series are missing, and I'm not sure why. The best I can figure is that I have constructed my measurements in a random sort of way that a developer who doesn't really understand the economics of "what goes with what" would do it, and that is revealing bugs(?) that won't show up when data elements are related in a sensible way. But I've banged my head against this long enough and would like to get it through.
